### PR TITLE
Exclude two generated classes from spotbugs

### PR DESCRIPTION
**Pull request by StefanSpieker**
Exclude two generated classes from spotbugs to fix errors reported in `Medium` scanning. It reports this for example:

```
public static String CannotBeInstalled(Object arg0, Object arg1, Object arg2) {
        return holder.format("CannotBeInstalled", arg0, arg1, arg2);
}
```
spotbugs reports: 

> Method names should start with a lower case letter
> Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.
> Bug kind and pattern: Nm - NM_METHOD_NAMING_CONVENTION

So I set them to ignore, better might be to rename the methods, but I did not find a way to generate it with a lower case.

### Proposed changelog entries

N/A

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [x] Proper changelog labels are set so that the changelog can be generated automatically
- [x] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [x] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: src/findbugs/findbugs-excludes.xml
Comment (timja) [inline src/findbugs/findbugs-excludes.xml:27]: to generate with lowercase you need to change the properties to start with a lowercase, e.g. https://github.com/jenkinsci/jenkins/blob/96a5526bdb1bad3f553d9a9e7f5987a1e6b11456/core/src/main/resources/hudson/tools/Messages.properties#L34

it would break any external localization though and doesn't really matter

**Review by timja — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

**Review by basil — APPROVED** [review]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Pull request merged**
